core/primitives/src/stateless_validation/state_witness.rs

```diff
@@ -146,13 +146,6 @@ pub struct ChunkStateWitness {
     /// After these are applied as well, we should arrive at the pre-state-root
     /// of the chunk that this witness is for.
     pub implicit_transitions: Vec<ChunkStateTransition>,
-    /// Finally, we need to be able to verify that the new transitions proposed
-    /// by the chunk (that this witness is for) are valid. For that, we need
-    /// the transactions as well as another partial storage (based on the
-    /// pre-state-root of this chunk) in order to verify that the sender
-    /// accounts have appropriate balances, access keys, nonces, etc.
-    pub new_transactions: Vec<SignedTransaction>,
```

Comment:
you cannot remove fields from state witness since it breaks borsh backward compatibility
I suggest renaming to `_new_transactions` and adding deprecation comment